soc/arm/st_stm32/stm32l0/power.c

```diff
@@ -41,12 +41,6 @@ void pm_state_set(enum pm_state state, uint8_t substate_id)
 		LL_LPM_EnableDeepSleep();
 		k_cpu_idle();
 		break;
-	case PM_STATE_SOFT_OFF:
-		LL_PWR_ClearFlag_WU();
-		LL_PWR_SetPowerMode(LL_PWR_MODE_STANDBY);
```

Comment:
Standby mode actually does imply context loss, so I believe it could be used as the poweroff method for the STM32 MCUs without Shutdown mode. I guess adding this state to the exit hook was a harmless mistake.

Reply:
So if I understand correctly, standby would be the way to poweroff L0 right?

Reply:
It's as close to power down as possible for L0 and many other STM32 series.